test/unit/rules/no-link-to-positional-params-test.js

```diff
@@ -28,23 +28,84 @@ generateRuleTests({
 
   bad: [
     {
-      template: '{{link-to "about"}}',
+      template: '{{link-to "About Us" "about"}}',
```

Comment:
I'd prefer that if we're changing the logic within this rule that we only add new test cases, not modify any existing ones. This helps us prevent regressions, since if we change the rule's logic and all the existing tests pass, we know we've preserved at minimum the existing functionality. 

Can we bring back the existing tests and just add new ones?

Reply:
Hi Steve, thanks for reviewing!

I agree with keeping existing tests to prevent regressions, I kept all the others, but in this case I find it strange to do so because:
- `{{link-to "about"}}` isn't valid use of the component, inline `{{link-to}}` components require _both a label and a route_
  - It doesn't compile in newer projects
  - It triggers a runtime error in older projects
  - We don't lint against `{{link-to}}` as well (no params at all, which also isn't valid use of the component)
- [The old logic assumed that the first string argument was always the route](https://github.com/ember-template-lint/ember-template-lint/blob/master/lib/rules/no-link-to-positional-params.js#L31-L33) which isn't true for inline `{{link-to}}` components (the label comes first), meaning the provided error message wasn't completely correct
- The [other](https://github.com/ember-template-lint/ember-template-lint/blob/master/test/unit/rules/inline-link-to-test.js) [link-to](https://github.com/ember-template-lint/ember-template-lint/blob/master/test/unit/rules/no-link-to-tagname-test.js) tests don't have this case either, I'm assuming also for the reasons mentioned above

Reply:
I added the test again, though I think it's strange to lint something that doesn't compile/run.

Reply:
Ah, so you're saying the inline version of `link-to` doesn't allow a single route param, correct? I think you're right - the inline version requires two params. Thanks for finding this. Let's change it to the correct params. Appreciate you fixing this!

Reply:
Correct, it needs 2 params to compile/run. So I'll delete this test case?

Reply:
Yep, sounds good!